corehq/apps/hqwebapp/static/hqwebapp/less/_hq/modals.less

```diff
@@ -125,3 +125,84 @@
         transform: translate(0, 0) !important;
     }
 }
+
+.add-new-module-modal {
+
+    .modal-header {
+        border-bottom: 1px solid lightgray;
+    }
+
+    .modal-footer {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        margin-top: -20px;
+        margin-bottom:20px;
+        color: #676B70;
+    }
+
+    .modal-flex-container {
+        max-height: 400px;
+        // Make the container scrollable
+        overflow-y: auto;
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        justify-content: flex-start;
+        padding-top: 10px;
+        align-items: stretch;
+    }
+
+    .new-module-option {
```

Comment:
App manager has custom stylesheets, I think this should go in there.

Reply:
Moved in f618066a49c4d8ca4b0e2e3aa2e76de5fa535297